include/config.h.in~

```diff
@@ -0,0 +1,103 @@
+/* include/config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
Very much so! I don't have such an ignore rule because I always configure my editors not to leave that junk lying around -- I think some part of autotools is producing this particular one, since it keeps coming back.

(I have a `~/.gitignore` where I put stuff like `.*.sw?` (because vim) -- set `excludesfile` under `[core]` in your `~/.gitconfig` to `/home/xyzzy/.gitignore` and it'll apply all the time!)

Reply:
Yeah, looking at my own `~/.gitignore` I see I've already got it ignoring Vim's `.*.swp` backups, so this must indeed come out of autotools.  Sigh.